.github/workflows/test_prs.yml

```diff
@@ -12,7 +12,7 @@ jobs:
       FLASK_APP: OpenOversight.app
     strategy:
       matrix:
-        python-version: ["3.11", "3.12"]
+        python-version: ["3.11", "3.12", "3.13-rc"]
```

Comment:
Basing the testing on this container: https://github.com/docker-library/python/blob/ec14acd6a0830012b4330429bc173247a375857e/3.13-rc/slim-bullseye/Dockerfile